.changeset/stupid-walls-glow.md

```diff
@@ -0,0 +1,5 @@
+---
+"stylelint": patch
+---
+
+Fixed: `selector-no-qualifying-type` reported ranges
```

Comment:
```suggestion
Fixed: `selector-no-qualifying-type` end positions
```

I prefer your wording but it's not consistent with the past changesets.

Reply:
This changes fixes the entirety of the reported ranges, as raised in https://github.com/stylelint/stylelint/issues/7904

While https://github.com/stylelint/stylelint/issues/6234 was only about the end positions.

The current changelog entry best describes both fixes.